tool/utils.ts

```diff
@@ -16,10 +16,15 @@ const fetch = initMakeFetchHappen();
 
 shell.config.fatal = true;
 
-// The current platform's operating system. Throws if the operating system
+// Directory that holds source files.
+const BUILD_PATH = 'build';
+
+export type OSKind = 'linux' | 'macos' | 'windows';
```

Comment:
It's probably a good idea to specify that this is specifically *Dart's* notion of OS kind, that it's different than Node's, and that you can convert from Node's to Dart's with `getOS()`. You could even name this something like `DartPlatform` to make it clearer.

Similarly for `ArchKind` below.